### ISSUE TITLE
Strings only available in englisch

### ISSUE THREAD

**Issue by de-es** [haiwen/seahub#4289]
There are several strings in Seahub which will be still displayed in English or disappear when changing the language setting. (7.0.5 CE / 7.0.9 PE).

This isn't a translation problem per se because these translations already exist in Transifex and in the .po/.mo files. I guess they were just not ported to the React interface.

These are the strings I found for my use case.

```
"You don't have any share links"
"You don't have any favorite items yet"
"You have not shared any folders"
"Notifications" (as title tooltip)
```

**Markdown Editor**

```
"Editing files in this browser can lead to slight display problems." (message popup)
"Insert file" (title)
"Open parent directory"
"Show side panel"
"Hide side panel"
"Size"
"Location"
"Last Update"
"Tags"
"Related Files"
"No document improvement suggestion"
"Show resolved comments"
```

**Comment by freeplant** [comment]
We will check the problem.

**Comment by JIMhackKING** [comment]
The problem does exist, and some sections are not translated

**Comment by freeplant** [comment]
Fixed in 7.1 version.
